selenium_tests/base_selenium_test_case.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+from selenium import webdriver
+from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
+from pyvirtualdisplay import Display
+from django.conf import settings
+
+
+# Enabling stdout logging only for high `-v`
+class BaseSeleniumTestCase(unittest.TestCase):
+    driver = None  # make selenium driver available to any part of the test case
+    display = None  # a reference to the virtual display (for running tests locally)
+
+    @classmethod
+    def setUpClass(cls):
+        """
+        Sets up the test case, including the selenium browser driver to use
+        """
+
+        local = True
```

Comment:
This should use the SELENIUM_CONFIG variable now available. Some of the comments in this file can be removed too (see the most recent updates to this file in Hiu Kei's PR), but that's not important.